src/main/java/com/teragrep/pth_06/config/HdfsConfig.java

```diff
@@ -90,7 +90,7 @@ public HdfsConfig(Map<String, String> opts) {
                 );
         hdfsPath = getOrThrow(opts, "hdfs.hdfsPath");
         hdfsUri = getOrThrow(opts, "hdfs.hdfsUri");
-        UseHdfsHostname = getOrThrow(opts, "hdfs.UseHostName");
+        useHdfsHostname = getOrThrow(opts, "hdfs.UseHostName");
```

Comment:
the config key uses capitalized `U`, is it intentional?

Reply:
Unintentional.
Solved in commit 149201a5a16d65e97ea767f6e6afdbeb3e3c1b6e